### DISCUSSION TITLE
Stereo SBS 180 image/video. How can be shown in the XR environment StereoKit.

### DISCUSSION THREAD

**Discussion by salexx** [StereoKit/StereoKit#925] [Q&A] [answered]
Stereo SBS 180 image/video. How can be shown in the XR environment StereoKit?
For example, is this an SBS image at 180 degrees.

![stereo-SBS-180](https://github.com/StereoKit/StereoKit/assets/3436243/04c9c9b4-bcdf-44e3-b2de-0cb150161fd9)

**Comment by maluoi** [comment, marked as answer]
Try using this shader on half of a UV sphere!
https://gist.github.com/maluoi/a050dfcb2387034989ad24c198829162

**Comment by salexx** [comment]
**Thank you!!!** I just started studying your engine today. I really liked him.
I was able to make a hemisphere and apply your shader. Everything **works great!!!**

Where can I get a stereo shader for a flat SBS image?
I understand that I need to change your shader a little. But I don't have enough knowledge yet (

Example of a flat SBS image:
![flatSBS](https://github.com/StereoKit/StereoKit/assets/3436243/8296c592-79aa-4b59-aba1-4215915e9396)

**Reply by salexx** [reply]
Sorry, I made a mistake somewhere. The flat images I need work beautifully with this shader.

**Thanks again for your project!!!**

**Reply by maluoi** [reply]
Glad you're liking it! :)

Yeah, I originally designed that shader while testing flatter images like these, so I'm glad it worked well on the 180 format too!

**Comment by salexx** [comment]
Model of a hemisphere with normals inward and correct UV
Making a hemisphere is not difficult. UV coordinates are more difficult to make)

You may want to embed this model into StereoKit)

[halfdome-SK.zip](https://github.com/StereoKit/StereoKit/files/14802600/halfdome-SK.zip)
